classes/mcPy/MultiProcessing.py

```diff
@@ -58,7 +58,7 @@ def add_task(self, func, args: list, **kwargs):
         try:
             self.TASK_LIST.put_nowait(data)
         except Full:
-            logging.warning()
+            logging.warning(Full)
```

Comment:
Good catch, though we might want to make this a little cleaner (add a better message than just the error)